tests/php/Functional/WC/Payment/Mollie_WC_Payment_Order_Test.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Mollie\WooCommerceTests\Functional\Payment;
+
+use Mollie\Api\Resources\Order;
+use Mollie\WooCommerceTests\TestCase;
+
+use Mollie_WC_Helper_Data;
+use Mollie_WC_Payment_Order;
+use Mollie_WC_Payment_OrderItemsRefunder;
+use WC_Order;
+
+use function Brain\Monkey\Functions\when;
+
+
+class Mollie_WC_Payment_Order_Test extends TestCase
+{
+    /* -----------------------------------------------------------------
+       onWebhookCanceled Tests
+       -------------------------------------------------------------- */
+
+    /**
+     * given onWebhookCanceled is called
+     * when order has status of Completed|Refunded|Cancelled
+     * then status does NOT change and we exit the method.
+     *
+     * @test
+     */
+    public function onWebHookCanceled_Returns_whenFinalStatus(){
+        /*
+        * Setup Stubs
+        */
+        $orderItemsRefunded = $this->buildTesteeMock(
+            Mollie_WC_Payment_OrderItemsRefunder::class,
+            [],
+            []
+        )->getMock();
```

Comment:
When you do unit tests you're mocking everything, so mocks can be reused on other tests too.

If you define a method which return a mock instance you can just call that method when you need in the test, this will make the test method body shorter.